app/views/content/welcome.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Welcome to teaching
+draft: true
```

Comment:
Does this mean we should build in removing/reverting the env variables, which you added as a separate PR yesterday, to this PR? @peteryates

Reply:
No, this will just stop the pages from appearing in the sitemap.

The if statement in the welcome guide will prevent the page from rendering if someone manually navigates to `/welcome`

Thinking about it, it might be worth preventing `draft` pages from being rendered at all (or being searchable) in production, but that's a bigger change and would be a separate PR.